php-template/UniverseTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+    include 'Universe.php';
+    function test() {
+        if (countAllStars([2,3]) ==  5 && countAllStars([9,-3]) == 6) {
+
+            echo("TECHIO> success true \r\n");
+            $file = file_get_contents("./Universe.php");
```

Comment:
Maybe we can use rename_function/override_function but we will need to install PECL package. I'll let @sidsharma27 answer that.